CountriesOfTheWorld/CountriesOfTheWorld/Extensions/UIStackViewExtensions.swift

```diff
@@ -0,0 +1,18 @@
+//
+//  UIStackViewExtensions.swift
+//  CountriesOfTheWorld
+//
+//  Created by Khrystyna Matasova on 14.12.2021.
+//
+
+import Foundation
+import UIKit
+
+extension UIStackView {
+
+    func removeAllArrangedSubviews() {
+        for subview in self.subviews {
+               subview.removeFromSuperview()
+           }
+    }
```

Comment:
```suggestion
    func removeAllSubviews() {
        self.subviews.forEach { $0.removeFromSuperview }
    }
```

UIStackView has difference between arranged subview and subviews, so naming is not 100% correct. Also you could simplify your code.
JFYI.

Reply:
wow, thank you!